packages/adelus/src/BlasWrapper_copy.hpp

```diff
@@ -0,0 +1,71 @@
+#ifndef BLASWRAPPER_COPY_HPP_
+#define BLASWRAPPER_COPY_HPP_
+
+#include<BlasWrapper_copy_spec.hpp>
+#include<KokkosKernels_helpers.hpp>
+
+namespace BlasWrapper {
```

Comment:
Should this also be in the Adelus namespace or is this in a Kokkos namespace?

Reply:
When copy and iamax are included in Kokkos Kernels, I will  remove these BlasWrapper* files and use KokkosKernels interfaces instead. These are just for now.

Reply:
Should I change to Adelus namespace?

Reply:
I think it is fine to leave them for now if you plan on moving them later.  I didn't know if they were permanent to Adelus or not.

Reply:
Thanks. I am leaving it for now.